mlrun/datastore/base.py

```diff
@@ -147,6 +147,10 @@ def _get_secret(self, key: str, default=None):
     def url(self):
         return f"{self.kind}://{self.endpoint}"
 
+    @property
+    def spark_url(self):
+        return self.url
+
```

Comment:
Looks like redundant to mlrun.datastore.utis.store_path_to_spark(). No need to introduce a new API in my opinion

Reply:
The suggestion being that we add a case to replace `webhdfs://` with `hdfs://` in `store_path_to_spark()`?

Reply:
That can be done, but the issue is the different port.

Reply:
This is should be like in S3: You work with url without the hostname/port :
val df = spark.read.text("hdfs:///user/hadoop/kuku.txt")

BUT: in the spark options you add {"spark.hadoop.fs.defaultFS":"hdfs://localhost:9000"}